src/App.jsx

```diff
@@ -1,13 +1,25 @@
+import { useState } from 'react';
+
 import Footer from './components/Footer';
 import Header from './components/Header';
 import Home from './components/Home';
+import { refreshAccessToken } from './common/auth';
+
+const App = () => {
+  const isUserLogged = !!JSON.parse(sessionStorage.getItem('token'));
+  const [isLogged, setIsLogged] = useState(isUserLogged);
```

Comment:
Is it worth to just make initial auth request on the first load and get rid of the button?